src/include/sof/lib/dai-legacy.h

```diff
@@ -190,6 +190,9 @@ struct dai_data {
 	 * the SOF_DAI_CONFIG_FLAGS_PAUSE flag.
 	 */
 	bool delayed_dma_stop;
+
+	/* ipc version special data */
+	void *private;
```

Comment:
Looks like we are always casting to the same type llp_slot_info ?

Reply:
yes, it is for IPC4. IPC3 may use it differently or even not use it